nuitka/optimizations/Optimization.py

```diff
@@ -59,10 +57,12 @@ def signalChange(tags, source_ref, message):
     """ Indicate a change to the optimization framework.
 
     """
+    from nuitka import Options
+
     if message is not None:
         # Try hard to not call a delayed evaluation of node descriptions.
 
-        if _is_verbose:
+        if Options.isVerbose():
```

Comment:
This one is however potentially run a lot of times, for every optimization Nuitka makes. Making a Python function call is unacceptable in this cases, there is nothing slower in Python than that.

Reply:
yeah, i don't like neither... how to decouple Options in that case then? Maybe a better logging system in the first place to be configured upfront? Anyway, as i said, consider these set of changes as an intermediate step for future PRs improving it

Reply:
You are repeating something about a future that seems awfully unclear.

And no, another logging system is not the solution. To reduce the overhead, this is the best solution short of having function variants depending on values. That again would definitely be not needed. One truth test of a global variable is not that bad.